cli/remote_build.go

```diff
@@ -272,22 +235,15 @@ func (p *RemoteCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}
 		// Apply changes that weren't committed yet
 		worktree, err := workRepo.Worktree() // current worktree
 		if err != nil {
-			patchErrored()
 			log.Errorf("Couldn't get current worktree: %v", err)
 			return subcommands.ExitFailure
 		}
 
-		if log.CheckIfTerminal() {
-			patchProgress = plumbing.NewProgressSpinner("Generating patch for local changes")
-			patchProgress.Start()
-		} else {
-			log.Info("Generating patch for local changes")
-		}
+		log.Info("Generating patch for local changes")
```

Comment:
Just for consistency with the above output:
```suggestion
		log.Info("Generating patch for local changes...")
```